items/items.js

```diff
@@ -52,6 +51,27 @@ const managedItemProvider = osgi.getService('org.openhab.core.items.ManagedItemP
  */
 const DYNAMIC_ITEM_TAG = '_DYNAMIC_';
 
+/**
+ * Helper function to convert JS types to openHAB command/state types' string representation.
+ *
+ * Numbers and strings are passed.
+ * Objects should implement `toOpenHabString` (prioritized) or `toString` to return a openHAB compatible representation.
+ *
+ * @private
+ * @param {*} value
+ * @returns {*}
+ */
+const _toOpenhabString = (value) => {
+  if (typeof value === 'number' || typeof value === 'string') {
```

Comment:
Since the function returns value by default do you need to check for number or string here?

Reply:
If we want to pass though number and don’t call `toString` on numbers, we need to check at least for number here.
`sendCommand` and `postUpdate` accept both number, string and openHAB types (see https://www.openhab.org/javadoc/latest/org/openhab/core/model/script/actions/busevent). 

I think converting a number to string can lead to loss of precision, I also wonder if it would be better to pass JS Quantity as QuantityType instead of its string representation. WDYT?

Reply:
> If we want to pass though number and don’t call toString on numbers,

Gotcha , i missed that `value.toString === 'function'` would match a number

> I also wonder if it would be better to pass JS Quantity as QuantityType instead of its string representation. WDYT?

I so rarely use UoM that i'm cautious to give an opinion one way or another.

Reply:
Let's leave as it is and merge :+1: